TransformationSystem/DB/TransformationDB.sql

```diff
@@ -1,27 +1,27 @@
 -- -------------------------------------------------------------------------------
---  Schema definition for the TransformationDB database a generic
---  engine to define input data streams and support dynamic data
---  grouping per unit of execution.
+-- Schema definition for the TransformationDB database a generic
+-- engine to define input data streams and support dynamic data
+-- grouping per unit of execution.
 
--- When installing via dirac tools, the following is not needed (still here for reference)
+-- When installing via dirac tools, the following is not needed(still here for reference)
 --
--- DROP DATABASE IF EXISTS TransformationDB;
--- CREATE DATABASE TransformationDB;
+-- DROP DATABASE IF EXISTS TransformationDB
+-- CREATE DATABASE TransformationDB
 -- ------------------------------------------------------------------------------
 -- Database owner definition
--- USE mysql;
+-- USE mysql
 -- Must set passwords for database user by replacing "must_be_set".
--- GRANT SELECT,INSERT,LOCK TABLES,UPDATE,DELETE,CREATE,DROP,ALTER,REFERENCES ON TransformationDB.* TO Dirac@'%' IDENTIFIED BY 'must_be_set';
--- FLUSH PRIVILEGES;
+-- GRANT SELECT, INSERT, LOCK TABLES, UPDATE, DELETE, CREATE, DROP, ALTER, REFERENCES ON TransformationDB.* TO Dirac@'%' IDENTIFIED BY 'must_be_set'
+-- FLUSH PRIVILEGES
 
 -- -----------------------------------------------------------------------------
-USE TransformationDB;
+USE TransformationDB
```

Comment:
I don't think the removal of the semicolons in this SQL file was fully intentional, was it?

Reply:
Yes exactly. Just corrected: 1f64b28718c2cb62c6508eae344d2f3cc780bcc8
Thank you.